### PR TITLE
Tags

### PR DESCRIPTION
**Pull request by ivanms1**
Tags for projects
Use yarn in the frontend

### CODE REVIEW

File: client/src/components/Cardv2/CardDetails/CardDetails.js [hunk 1]
```diff
@@ -161,6 +169,12 @@ export const CardDetails = ({}) => {
                   {getCurrentDate(project.createdAt)}
                 </span>
 
+                <div className='tagsContainer'>
```
Comment (ivanms1) [inline client/src/components/Cardv2/CardDetails/CardDetails.js:172]: use styled components like in the rest of the app.

Comment (ivanms1) [inline client/src/components/Cardv2/CardDetails/CardDetails.js:172]: try to stick with styled components, or target by tag if already inside a styled component.

File: client/src/pages/SubmitYourProject/SubmitForm.js [hunk 2]
```diff
@@ -174,6 +164,22 @@ function SubmitForm({ user, onSubmit }) {
...
+            name='tags'
+            control={control}
+            render={({ onChange }) => (
+              <SelectTags name='tags' onChange={onChange} options={options} />
+            )}
+            rules={{ required: 'Tags cannot be Empty' }}
+            defaultValue=''
+          />
+          <ErrorMessage errors={errors} name='tags' as={<ErrorText />}>
+            {(errors) => <small>{console.log(errors)}</small>}
```
Comment (ivanms1) [inline client/src/pages/SubmitYourProject/SubmitForm.js]: console.log?

Comment (Raj2631) [reply client/src/pages/SubmitYourProject/SubmitForm.js]: ah mb, fixed

**Review by ivanms1 — COMMENTED** [review]
@Raj2631 @revengemiroz someone approve this lol

**Pull request merged**